modules/front-end/src/styles/nz-select.less

```diff
@@ -66,12 +68,24 @@ nz-select:not(.ant-select-customize-input) {
     }
 
     &.nz-select-36 .ant-select-selector {
-      height: 36px;
+      min-height: 36px;
+      max-height: 100px;
+      padding-top: 0;
+      padding-bottom: 0;
+      overflow-x: scroll;
```

Comment:
This line add a horizontal scroll bar permannently, should remove this line

Reply:
Have fixed